frontend/src/metabase-lib/display.ts

```diff
@@ -0,0 +1,160 @@
+import * as Lib from "metabase-lib";
+import type {
+  CardDisplayType,
+  VisualizationSettings,
+} from "metabase-types/api";
+import { TYPE } from "metabase-lib/types/constants";
+import { isa } from "metabase-lib/types/utils/isa";
+
+type DefaultDisplay = {
+  display: CardDisplayType;
+  settings?: Partial<VisualizationSettings>;
```

Comment:
Is `map` the only display type that has settings?

Reply:
AFAIK every display type can have settings, even "table" (e.g. conditional formatting)